src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/mappers/TripPatternForDateMapper.java

```diff
@@ -72,7 +71,7 @@ public TripPatternForDate map(Timetable timetable, LocalDate serviceDate) {
       if (!serviceCodesRunning.contains(tripTimes.getServiceCode())) {
         continue;
       }
-      if (tripTimes.getRealTimeState() == RealTimeState.CANCELED) {
+      if (tripTimes.isDeleted()) {
```

Comment:
Shouldn't we exclude both CANCELLED and DELETED here?

Reply:
No, since those will be either filtered or not in the `RaptorRoutingRequestTransitData`.